### DISCUSSION TITLE
Database integration/Teneo

### DISCUSSION THREAD

**Discussion by marcin-kaczor-ck** [eclipse-sirius/sirius-desktop#309] [Q&A]
I'm trying to setup/run project in newest ObeoDesigner with database integration using teneo, but can't install plugin
using update site: 
https://download.eclipse.org/modeling/emf/teneo/updates/2.1.0/interim/
getting error: 
Cannot complete the install because one or more required items could not be found.
  Software being installed: EMF Teneo Hibernate SDK 2.1.0.v201702222200 (org.eclipse.emf.teneo.hibernate.sdk.feature.group 2.1.0.v201702222200)
  Missing requirement: EMF Teneo Hibernate 2.1.0.v201702222200 (org.eclipse.emf.teneo.hibernate 2.1.0.v201702222200) requires 'java.package; javax.persistence.criteria [2.0.0,3.0.0)' but it could not be found
  Cannot satisfy dependency:
    From: EMF Teneo Hibernate Runtime 2.1.0.v201702222200 (org.eclipse.emf.teneo.hibernate.feature.group 2.1.0.v201702222200)
    To: org.eclipse.equinox.p2.iu; org.eclipse.emf.teneo.hibernate [2.1.0.v201702222200,2.1.0.v201702222200]
  Cannot satisfy dependency:
    From: EMF Teneo Hibernate SDK 2.1.0.v201702222200 (org.eclipse.emf.teneo.hibernate.sdk.feature.group 2.1.0.v201702222200)
    To: org.eclipse.equinox.p2.iu; org.eclipse.emf.teneo.hibernate.feature.group [2.1.0.v201702222200,2.1.0.v201702222200]
I'm able to install Teneo in "normal" eclipse. 
Is there any alternative to this?

**Comment by pcdavid** [comment]
From the error message, I'd say Teneo might be too old to work in recent environments where some `javax.*` packages have become `jakarta.*`, but I'm not familiar with it so it's just a guess.

When you say "normal Eclipse", which version and which package are you using?

**Comment by marcin-kaczor-ck** [comment]
Normal Eclipse means 2023-12, I can install there Teneo and then Obeo plugin will uninstall it, so I can't make them work together.
